amundsen_application/api/utils/request_utils.py

```diff
@@ -43,15 +44,17 @@ def request_metadata(*,     # type: ignore
                            client=app.config['METADATASERVICE_REQUEST_CLIENT'],
                            headers=headers,
                            timeout_sec=timeout_sec,
-                           data=data)
+                           data=data,
```

Comment:
any reason we can't use data but need another json input param?

Reply:
For json input, we can pass dict directly. I try to use `json.dumps(dict)` combined with data,  but seems that it can't be recognized by search service.